frontend/src/components/DownloadAnalysisLink/DownloadAnalysisLink.tsx

```diff
@@ -15,28 +17,25 @@ export let DownloadAnalysisLink = withStore()(({ store }) =>
     icon={<DownloadIcon />}
     label='Download'
     labelPosition='before'
-    disabled={isEmpty(store.get('adequacies'))}
     onClick={onClick(store)}
   />
 )
 DownloadAnalysisLink.displayName = 'DownloadAnalysisLink'
 
 function onClick(store: Store) {
+  let method = store.get('method')
   return () => {
     let headers = [
       'CountyName',
       'ZipCode',
       'SpecDesc',
-      'BeneficiariesWith15MileAccess',
-      'BeneficiariesWith30MileAccess',
-      'BeneficiariesWith50MileAccess',
-      'BeneficiariesOver60MileAccess',
-      'MinDist (mi)',
-      'AvgDist (mi)',
-      'MaxDist (mi)',
-      'MinTime (min)',
-      'AvgTime (min)',
-      'MaxTime (min)'
+      getLegend(method, AdequacyMode.ADEQUATE_15),
+      getLegend(method, AdequacyMode.ADEQUATE_30),
+      getLegend(method, AdequacyMode.ADEQUATE_60),
+      getLegend(method, AdequacyMode.INADEQUATE),
+      'Min ' + method,
+      'Avg ' + method,
```

Comment:
Curious how the average will look with `ABSURDLY_LARGE_TIME_IN_MINUTES` 😂

Reply:
Should I remove? I also agree that it doesnt make much sense